### PR TITLE
Fix QueueHipRtBlocking not being blocking

### PR DESCRIPTION
**Pull request by BenjaminW3**
Follow-up to #877

I had to update ROC from 2.7.0 to 2.8.0 because this is the first version
which supports hipMemset3DAsync.

### CODE REVIEW

**Comment by psychocoderHPC** [comment]
> I had to update ROC from 2.7.0 to 2.8.0 because this is the first version

np I was only on the way to do it. Since the AMD ecosystem is currently changing fast I see no problem to enforce newer versions and skip older versions.

**Review by psychocoderHPC — APPROVED** [review]

File: include/alpaka/mem/buf/hip/Copy.hpp [hunk 1]
```diff
@@ -993,7 +1000,11 @@ namespace alpaka
                     // Initiate the memory copy.
                     ALPAKA_HIP_RT_CHECK(
                         hipMemcpy3D(
```
Comment (psychocoderHPC) [inline include/alpaka/mem/buf/hip/Copy.hpp]: must be `hipMemcpy3DAsync`

**Review by psychocoderHPC — DISMISSED** [review]

File: include/alpaka/mem/buf/hip/Copy.hpp (hunk 1 above)
Comment (BenjaminW3) [reply include/alpaka/mem/buf/hip/Copy.hpp]: fixed

**Comment by BenjaminW3** [comment]
Ready to be approved/merged!

**Review by psychocoderHPC — APPROVED** [review]

**Pull request merged**